### ISSUE TITLE
Work saving as a 0 byte file

### ISSUE THREAD

**Issue by BiancaG7** [photopea/photopea#4465]
I was working on a project for multiple hours on photopea, and i hit save every now and then. I was gonna stop working for a bit so I wanted to make sure my stuff wouldnt delete so I clicked "Save as PSD" so that it saves a file on my computer. So I trusted it turned off my computer. When I came back, I went to open the file and it kept saying " Unknown file format: "\u0000\u0000\u0000\u0000" ".
So i opened my computer file and found that it said "0 bytes" (so the file is empty). I tried opening it elsewhere and it said "File is empty"
So basically my work vanished into thin air.
I even went back to an older version of the same file which i had emailed myself and redownloaded that one. But it also downloaded as 0 bytes so now i dont know what to do. My work is lost and i had a due date

**Comment by photopea** [comment]
Can you create an empty document and save a PSD right now? Is it also 0 Bytes? If not, can you find any steps how to produce a 0-Byte file?

Usually, if you close Photopea before the "saving ..." message goes away (and saving does not finish), you will get a 0 Byte file.

**Comment by BiancaG7** [comment]
do you mean a photopea document?

**Comment by BiancaG7** [comment]
ok so i opened a new photopea file and left it completely empty/blank, saved it and it saved with 110 KB not 0

**Comment by photopea** [comment]
What you discribe has never happened to me. And whenever I try, a file is never 0 Bytes.

I really think you closed Photopea before the "saving ..." message goes away. You should always wait until the saving is complete. Saving a 200 MB PSD can take several seconds, it is not saved instantly.

**Comment by LinuXP15** [comment]
happens to me a lot, maybe my layers are too big or idk, but sometimes after a l;ong work layers or elements get currapted and seems partly transparent or missbehave, etc, but i camwe back to my pc this morning and when i woke it up the project seemed to be back to nornal and I even could save it, as preaviously it was stuck on "saving" and the file created stayed 0kb.

**Comment by photopea** [comment]
This mostly happens if you close Photopea while the message "Saving ..." is still on your screen. If your PSD file is 500 MB, and your storage can write 25 MB/s, saving will take 20 seconds. If you close Photopea after 10 seconds of saving, your file will be empty.

If you can make Photopea create a 0-Byte file, please, record a short video of how you do it. I am ready to pay you for finding a bug.

**Comment by LinuXP15** [comment]
i always wait for it to finish saving, but multiple times the layers start to glitch and saving to rescue the whole project malfunctions too. I can try to send projects where it happend and tell you what I did that made it start glitching and how, also preventing me from saving normally again.

**Comment by photopea** [comment]
I would be rather if you recorded a short video of how you make Photopea glitch. The glitch could depend on your enivronment (browser, operating system, browser extensions, the amount of RAM memory, etc) and I might not be able to make it happen in my environment.

**Comment by LinuXP15** [comment]
edg, ubo, w11, 4GB

sry for writing short

**Comment by photopea** [comment]
Is "ubo" Ublock Origin? Does the problem happen when you turn it off?

**Comment by LinuXP15** [comment]
I haven't tried but ubo is kinda a must

**Comment by photopea** [comment]
Okay. Maybe try to convince the creators of your extension to not change how Photopea works.

**Comment by LinuXP15** [comment]
i dont think adblocking is the reason, as i can both save while using it,
only until it starts glitching, and the only point of the addon is to
disable ads showing, unless my layers are somehow ads, nothing to worry
about ubo

‫בתאריך יום ה׳, 9 באוק׳ 2025 ב-13:25 מאת ‪Photopea‬‏ <‪
***@***.***‬‏>:‬

**Comment by photopea** [comment]
Do you have any problems when using Photopea with no browser extensions?

**Comment by LinuXP15** [comment]
I will check it

**Comment by LinuXP15** [comment]
i tried to reproduce the bug and didnt succeed, no that I have disabled ubo, i just tried to mess the project by playing with layers again, but photopea seemes to work well, god bless you

**Comment by photopea** [comment]
When I use Photopea with uBlock Origin, the free Transform stops working after a while. So do not expect Photopea to work correctly until uBlock fixes it.

**Comment by LinuXP15** [comment]
I wonder why, usually it seems fine, if ubo does causes that then my apology, i am not a developer like either you or them, but I hope to create something useable like you has donated pp and they ubo, to the internet
